db/migrate/20190619134855_rename_user_id_in_schools.rb

```diff
@@ -0,0 +1,5 @@
+class RenameUserIdInSchools < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :schools, :user_id, :admin_id
```

Comment:
until it's merge into development / master i recommend you to update `db/migrate/20190619131949_add_user_id_to_schools.rb` migration :)